releasenotes/notes/move-es-version-vars-17c22cfd558a68d5.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
The repo_build role tries to interpolate these variables when indexing the elasticsearch role for apt packages.